ucp/_libs/core.pyx

```diff
@@ -422,10 +422,14 @@ class _Endpoint:
                     self._ctrl_tag_send,
                     pending_msg=self.pending_msg_list[-1]
                 )
-            except UCXError:
+            except UCXError as e:
+                log = "UCX Closing Error on worker %d\n%s" % (hex(self.uid), str(e))
+                logging.error(log)
                 pass  # The peer might already be shutting down
```

Comment:
I'm guess we can drop the `pass` now, but maybe we should move the comment above?